client/verta/verta/_internal_utils/_utils.py

```diff
@@ -260,19 +260,19 @@ def __len__(self):
 
         return total_records
 
-    def find(self, where):
+    def find(self, *args):
         """
-        Gets the results from this collection that match predicates `where`.
+        Gets the results from this collection that match input predicates.
 
-        A predicate in `where` is a string containing a simple boolean expression consisting of:
+        A predicate is a string containing a simple boolean expression consisting of:
 
             - a dot-delimited property such as ``metrics.accuracy``
             - a Python boolean operator such as ``>=``
             - a literal value such as ``.8``
 
         Parameters
         ----------
-        where : str or list of str
+        args : strs or a list of str
```

Comment:
```suggestion
        *args : strs
```

Reply:
Even though it takes a list, I think we should have an official preference on what to take

Reply:
Makes sense to me.

Reply:
Fixed.